var/spack/repos/builtin/packages/ramses/package.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

Comment:
Please update the copyright ending year here and in the rest of the packages.

```suggestion
# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
```

Reply:
Thanks I have updated the copyright dates